flathub.yaml

```diff
@@ -0,0 +1,11 @@
+- name: Add the flathub flatpak repository remote
+  community.general.flatpak_remote:
+    name: flathub
+    state: present
+    flatpakrepo_url: https://flathub.org/repo/flathub.flatpakrepo
+  become: true
+  tags: flathub
```

Comment:
if you run playbook with `--tags discord` (or any other flatpak's software) this wont execute